celery tasks tests failing

**Describe your environment** Describe any aspect of your environment relevant to the problem, including your Python version, [platform](https://docs.python.org/3/library/platform.html), version numbers of installed dependencies, information about your cloud hosting provider, etc. If you're reporting a problem with a specific version of a library in this repo, please check whether the problem has been fixed on main.

`tox -e py39-test-instrumentation-celery`
```
>       self.assertEqual(
            event.attributes[SpanAttributes.EXCEPTION_TYPE], "CustomError"
        )
E       AssertionError: 'tests.celery_test_tasks.CustomError' != 'CustomError'
E       - tests.celery_test_tasks.CustomError
E       + CustomError

instrumentation/opentelemetry-instrumentation-celery/tests/test_tasks.py:128: AssertionError
```
**Steps to reproduce**
Describe exactly how to reproduce the error. Include a code sample if applicable.
Run `tox -e py39-test-instrumentation-celery`

**What is the expected behavior?**
Pass the tests

**What is the actual behavior?**
1 test failed

**Additional context**

Add capture the fully qualified type name for raised exceptions in spans (https://github.com/open-telemetry/opentelemetry-python/pull/3837)

--- a/instrumentation/opentelemetry-instrumentation-celery/tests/test_tasks.py
+++ b/instrumentation/opentelemetry-instrumentation-celery/tests/test_tasks.py
@@ -125,8 +125,10 @@ class TestCeleryInstrumentation(TestBase):
 
         self.assertIn(SpanAttributes.EXCEPTION_STACKTRACE, event.attributes)
 
-        self.assertEqual(
-            event.attributes[SpanAttributes.EXCEPTION_TYPE], "CustomError"
+        # TODO: use plain assertEqual after 1.25 is released (https://github.com/open-telemetry/opentelemetry-python/pull/3837)
+        self.assertIn(
+            "CustomError",
+            event.attributes[SpanAttributes.EXCEPTION_TYPE]
         )
 
         self.assertEqual(

--- a/instrumentation/opentelemetry-instrumentation-celery/tests/test_tasks.py
+++ b/instrumentation/opentelemetry-instrumentation-celery/tests/test_tasks.py
@@ -127,8 +127,7 @@ class TestCeleryInstrumentation(TestBase):
 
         # TODO: use plain assertEqual after 1.25 is released (https://github.com/open-telemetry/opentelemetry-python/pull/3837)
         self.assertIn(
-            "CustomError",
-            event.attributes[SpanAttributes.EXCEPTION_TYPE]
+            "CustomError", event.attributes[SpanAttributes.EXCEPTION_TYPE]
         )
 
         self.assertEqual(

